### DISCUSSION TITLE
Transcription in a crowded room

### DISCUSSION THREAD

**Discussion by patchenm** [deepgram/community#423] [General help] [answered]
Does anyone have suggestions on how to best use Deepgram for transcription in a crowded room (for example an office with a bunch of workers separated by cubicles?)

What I've found is that Deepgram seems to do a good job of isolating the primary speakers voice and ignoring all the background speaking while the primary speaker is talking.

However, if the primary speaker pauses to take a breath or think and someone is talking in the background, then the background voices immediately start getting transcribed.

Not sure if there is an option to 'fingerprint' the primary speaker', but I was wondering how others are handling this issue.

**Comment by DamienDeepgram** [comment, marked as answer]
Diarization might help in this case so you can know which speaker is saying what

https://developers.deepgram.com/docs/diarization
